### PR TITLE
Added description for several repositories

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of https://github.com/artipie/artipie/issues/317 
Added description for several repositories

### CODE REVIEW

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one correction

File: src/main/resources/dashboard/repo.hbs [hunk 1]
```diff
@@ -189,6 +189,59 @@ conda config --set anaconda_upload yes
...
 command and build and upload packages with <code>conda build</code>, or, if the package is already build,
 use <code>anaconda upload</code> command to publish package to Artipie.
+{{/eq}}{{#eq type "npm"}}
+To install or publish <code>npm</code> package into Artipie repository, specify the repository url
+with the <code>--registry</code> option:
+<pre>
+    npm publish --registry=https://central.artipie.com/{{user}}/{{name}}
+    npm install --registry=https://central.artipie.com/{{user}}/{{name}} <package name>
+</pre>
+{{/eq}}{{#eq type "npm"}}
```
Comment (genryxy) [inline src/main/resources/dashboard/repo.hbs:199]: here should be `gem`

**Comment by olenagerasimova** [comment]
@genryxy thanks

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Comment by olenagerasimova** [comment]
@g4s8 please merge

**Pull request merged**